int16 mode
int16 stage_bm
int16 tracking_status
int16[] lidar_levels
geometry_msgs/Quaternion coor
RouteNode nd_training
string graph
RouteNode nd_ocp
RouteNode nd_target
RouteNode[] work_list


